fabric/state.py

```diff
@@ -224,6 +224,13 @@ def _rc_path():
         help="default to parallel execution method"
     ),
 
+    make_option('--parallel-exit-on-errors',
```

Comment:
Thanks!

Em 30/07/2016 01:12, "Aravind Gottipati" <notifications@github.com>
escreveu:

> In fabric/state.py
> <https://github.com/fabric/fabric/pull/1471#discussion_r72873304>:
>
> > @@ -224,6 +224,13 @@ def _rc_path():
> >          help="default to parallel execution method"
> >      ),
> >
> > +    make_option('--parallel-exit-on-errors',
>
> yeah, I will map it to -E, and submit a patch.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/fabric/fabric/pull/1471/files/9703ed60af878ab8dd3c1a08a0a5b305428058ca#r72873304>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA3MAYspf3gdSTzh5jf-Pi9y8Jdghtmmks5qaojogaJpZM4Is-az>
> .
>